tests/test_mcp_toolset.py

```
@@ -0,0 +1,101 @@
+
+def test_mcpserver_unreachable():
+    mcp_toolset = RemoteMCPToolset(
+        url="http://0.0.0.0:3009",
+        name="test_mcp",
+        description="",
+    )
+
+    assert (
+        False,
+        "Failed to load mcp server test_mcp http://0.0.0.0:3009/sse ('unhandled errors in a TaskGroup', [ConnectError('All connection attempts failed')])",
+    ) == mcp_toolset.init_server_tools(config=None)
```

Ruff (0.11.9)

Yoda condition detected

(SIM300)